py/htm/examples/rest/rest_api.py

```diff
@@ -0,0 +1,211 @@
+from urllib.parse import urlencode
+import requests
+import json
+
+
+class NetworkError(Exception):
+  pass
+
+
+def process_response(rsp):
+  if rsp.status_code != requests.codes.ok:
+    raise NetworkError('HTTP Error')
+
+  text = rsp.text.strip()
+
+  if text[0:6] == 'ERROR:':
+    raise NetworkError(text[6:].strip())
+
+  if text == 'OK':
+    return text
+
+  idx = text.find('[')
+
+  if idx == -1:
+    return text
+
+  data = text[idx + 1:].replace(']}', '')
+
+  return json.loads('[' + data + ']')
+
+
+class Network(object):
```

Comment:
this helper class indeed simplifies the usage of the REST API significantly :+1: If it's generic enough for use, we should place it in a proper place - `/bindings/rest/`?
CC @dkeeney please weight in